.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,20 @@
+name: pre-commit
```

Comment:
This will add the pre-commit run as a GH-action. It helps to run those checks for those who don't have pre-commit installed.